docs/self-managed/operational-guides/update-guide/840-to-850.md

```diff
@@ -0,0 +1,21 @@
+---
+id: 840-to-850
+title: Update 8.4 to 8.5
+description: "Review which adjustments must be made to migrate from Camunda 8.4.x to Camunda 8.5.0."
+---
+
+The following sections explain which adjustments must be made to migrate from Camunda 8.4.x to 8.5.x for each component.
```

Comment:
@akeller I know we handled this in a previous discussion, but when do we write out versioning as `8.5.0` or `8.5.x`, for example? I can add this to my next WCoE release, but wanted to refresh my memory on this one.

Reply:
Based on how we describe updates [here](https://docs.camunda.io/docs/self-managed/operational-guides/update-guide/introduction/), "When updating from one minor version to the next, you do not need to update to each patch version along the way. However, do not skip minor versions when updating."

In this case, we could use 8.4.x to 8.5.x so the minor version is clear, but we don't need to be specific about the patch.